packages/nvidia_nat_weave/src/nat/plugins/weave/fastapi_plugin_worker.py

```
@@ -91,24 +101,40 @@ async def _add_weave_feedback_route(self, app: FastAPI, builder: WorkflowBuilder
+                        return feedback_added
 
                     try:
-                        await asyncio.to_thread(add_weave_feedback)
-                        return WeaveFeedbackResponse(
-                            message=f"Added reaction '{reaction_type}' to call {observability_trace_id}")
+                        feedback_added = await asyncio.to_thread(add_weave_feedback)
+                        feedback_str = " and ".join(feedback_added)
+                        return WeaveFeedbackResponse(message=f"Added {feedback_str} to call {observability_trace_id}")
                     except Exception as e:
                         logger.exception("Failed to add feedback to Weave")
                         raise HTTPException(status_code=500, detail=f"Failed to add feedback: {str(e)}") from e
```

Ruff (0.14.14)

[warning] Use explicit conversion flag

Replace with conversion flag

(RUF010)